src/service/Configurator_6D_Modes.tpp

```diff
@@ -0,0 +1,177 @@
+#ifndef SRC_CONFIGURATOR_6D_MODES_TPP_
+#define SRC_CONFIGURATOR_6D_MODES_TPP_
+
+#include <exception>
+#include <vector>
+
+#include "Configurator_6D_Modes.h"
+
+#include "readFile.h"
+#include "DOF_6D.h"
+//#include "ServerIncludes.h"
+#include "Server.h"
+#include "Protein.h"
+#include "GridUnion.h"
+#include "ParamTable.h"
+#include "SimParam.h"
+#include "TypeMap.h"
+#include "DataManager.h"
+#include "DOFTransform.h"
+#include "nativeTypesMath.h"
+#include "ServiceFactory.h"
+
+
+namespace as {
+
+template<typename SERVICE>
+void Configurator_6D_Modes<SERVICE>::init(CmdArgs const& args) noexcept {
+
+	/* load dataItems */
```

Comment:
Ok for now. But this seems to be a lot of code duplication (compare Configurator_6D.tpp). We need to overcome this soon. Can you create a ticket for that?!

Reply:
done